avc_codec_registration.src.html

```diff
@@ -104,7 +104,7 @@
 {{EncodedVideoChunkType/key}}, and the bitstream is in
 {{AvcBitstreamFormat/avc}} format, then the {{EncodedVideoChunk}} is expected to
 contain a primary coded picture that is an instantaneous decoding refresh (IDR)
-picture.
+picture, or an I-frame with a recovery point SEI message.
```

Comment:
This needs more detail on the specific SEI recovery point payload that can be expected. I.e., IIUC it's specifically an I frame with `recovery_frame_cnt == 0` and `exact_match_flag=1` in the SEI recovery payload.